lib/src/browser/disposable_browser.dart

```diff
@@ -204,6 +204,11 @@ class Disposable implements disposable_common.Disposable {
       _disposable.manageDisposer(disposer);
 
   @override
+  disposable_common.Disposable getManagedDisposable(
```

Comment:
But in those cases the method is acting as a factory, in this case it's just passing its argument through.